Source/NaturalLanguageUnderstandingV1/Models/CategoriesOptions.swift

```diff
@@ -33,18 +33,10 @@ public struct CategoriesOptions: Codable, Equatable {
      */
     public var limit: Int?
 
-    /**
-     Enter a [custom
-     model](https://cloud.ibm.com/docs/services/natural-language-understanding?topic=natural-language-understanding-customizing)
-     ID to override the standard categories model.
-     */
-    public var model: String?
-
```

Comment:
This looks like it will be a breaking change.  Is that okay?

Reply:
Yes, that's a good catch. I've added a hand edit to keep this member in the struct. This should be resolved now.